geonode/templates/search/_pagination.html

```diff
@@ -6,13 +6,24 @@
 <span ng-bind="numpages"></span>
 <a href><strong ng-click="paginate_up()">&raquo;</strong></a> -->
 
+
 <div class="row">
-  <nav>
-	<ul class="pagination pull-right">
-	  <li ng-click="paginate_down()"><a href><strong><i class="fa fa-angle-left"></i></strong></a></li>
-	  <li><a href>{% trans "page" %} <span ng-model="page" ng-bind="page"></span> {% trans "of" %} <span ng-bind="numpages"></span></a>
-	  </li>
-	  <li ng-click="paginate_up()"><a href><strong><i class="fa fa-angle-right"></i></strong></a></li>
-	</ul>
-  </nav>
+    <nav>
+        <ul class="pagination pull-right">
+            <li onclick="scroll_top()" ng-click="paginate_down()"><a href><strong><i
```

Comment:
this means that the page is scrolled top BEFORE the paginated request is triggered, right?

Reply:
onclick is started before ng-click, but the execution is async. even if I go with ng-click="paginate_down();scroll_top()""  and pagination take a while scroll will be made first

Reply:
synchronized this with promises

Reply:
of course, the right way to do it would be to do the scroll when the paginate response arrives. But I don't know if we're able to subscribe to any pagination events here....

Reply:
this is exactly what i did Promise(pagination request).then(scroll)

Reply:
ok I lost you're latest commit. So it's fine.